pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -265,6 +266,16 @@ type DiskSourceHost struct {
 	Port string `xml:"port,attr,omitempty"`
 }
 
+type BackingStore struct {
```

Comment:
Can you also add a mapping test, so that we have the expected xml outcome tested?

Reply:
On it as soon as I figure out the functional test (thanks for pointing that out, totally missed the tmpfs mount).